src/main/antlr4/com/github/palindromicity/syslog/dsl/generated/Rfc5424.g4

```diff
@@ -37,6 +37,10 @@ grammar Rfc5424;
 
 }
 
+ heroku_https_log_drain     : nonzero_digit digit* sp heroku_https_log_drain_msg;
+
```

Comment:
Since they specifically say that they do not support structured data, we should just omit it and keep the rule clear and simple

Reply:
Should I just distill it down to:
```
 heroku_https_log_drain : nonzero_digit digit* sp header (sp)? (msg)? #herokuHttpsMsg;
```